<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainW2</class>
 <widget class="QMainWindow" name="MainW2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gopal.lv Sudoku Studio</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel[hi='1'] {
background-color:rgb(254,228,117)
}
QLabel {
background-color:rgb(144,208,177)
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>311</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::DashLine</enum>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>32</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>9</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>32</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>9</number>
    </attribute>
   </widget>
   <widget class="QGroupBox" name="groupBG">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>0</y>
      <width>91</width>
      <height>125</height>
     </rect>
    </property>
    <property name="title">
     <string>Colors</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QRadioButton" name="rbOrig">
       <property name="text">
        <string>Orig</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbFix">
       <property name="text">
        <string>Fix</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbWork">
       <property name="text">
        <string>Work</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbClear">
       <property name="text">
        <string>clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbNOOP">
       <property name="text">
        <string>NOOP</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPlainTextEdit" name="txtExImp">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>130</y>
      <width>91</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans Mono</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="toolTip">
     <string>Left=send to board; Right=import from board</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QPlainTextEdit::NoWrap</enum>
    </property>
    <property name="overwriteMode">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbPush">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>280</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Push</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbPop">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>280</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Pop</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblStack">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>280</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>--</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>81</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="lblH0">
       <property name="text">
        <string>H0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblH1">
       <property name="text">
        <string>H1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblH2">
       <property name="text">
        <string>H2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblH3">
       <property name="text">
        <string>H3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblH4">
       <property name="text">
        <string>H4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblH5">
       <property name="text">
        <string>H5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblH6">
       <property name="text">
        <string>H6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblH7">
       <property name="text">
        <string>H7</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblH8">
       <property name="text">
        <string>H8</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>281</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="VerticalLabel" name="lblV0">
       <property name="text">
        <string>V0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="VerticalLabel" name="lblV1">
       <property name="text">
        <string>V1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="VerticalLabel" name="lblV2">
       <property name="text">
        <string>V2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="VerticalLabel" name="lblV3">
       <property name="text">
        <string>V3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="VerticalLabel" name="lblV4">
       <property name="text">
        <string>V4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="VerticalLabel" name="lblV5">
       <property name="text">
        <string>V5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="VerticalLabel" name="lblV6">
       <property name="text">
        <string>V6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="VerticalLabel" name="lblV7">
       <property name="text">
        <string>V7</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="VerticalLabel" name="lblV8">
       <property name="text">
        <string>V8</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>330</y>
      <width>231</width>
      <height>65</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLabel" name="lblQ4">
       <property name="text">
        <string>Q4</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="lblQ8">
       <property name="text">
        <string>Q8</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblQ3">
       <property name="text">
        <string>Q3</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="lblQ2">
       <property name="text">
        <string>Q2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblQ6">
       <property name="text">
        <string>Q6</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblQ0">
       <property name="text">
        <string>Q0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="lblQ7">
       <property name="text">
        <string>Q7</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lblQ1">
       <property name="text">
        <string>Q1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="lblQ5">
       <property name="text">
        <string>Q5</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="gHi">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>380</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="tb1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tb2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>2</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tb3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>3</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tb4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>4</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tb5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>5</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tb6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>6</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tb7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>7</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tb8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>8</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tb9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>9</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VerticalLabel</class>
   <extends>QLabel</extends>
   <header>src/verticallabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
